core/src/main/java/io/grpc/internal/ServiceConfigInterceptor.java

```diff
@@ -190,6 +201,7 @@ public String toString() {
           .toString();
     }
 
+    @SuppressWarnings("BetaApi") // Verify isn't all that beta
```

Comment:
extra nitpicky nit: "isn't all that beta" is a little...informal. The same wording is already used in several other places in the repo (thus proving it's part of guava's stable api ;)), so it looks fine here too

Reply:
Modified with formal comment.